index.html

```diff
@@ -137,66 +173,101 @@ <h2 class="cont-header">Professional Art Printing Data</h2>
+                        <h3 class="iconic">Language</h3>
+                    </div>
+                    <ul class="langlist">
+                        <li class="jss">Javascript</li>
+                        <li class="jss">Ruby</li>
+                        <li class="jss">Html</li>
+                        <li class="jss"> Css</li>
+                    </ul>
+                </div>
+                <div class="framecont">
+                    <div class="lang-heading"><img src="images/rectangle.png" alt="">
+                        <h3 class="iconic">Framework</h3>
+                    </div>
+                    <ul class="langlist">
+                        <li class="jss">Bootstrap</li>
+                        <li class="jss">Ruby on Rails</li>
+                        <li class="jss">RSPec</li>
+                        <li class="jss">CapyBara</li>
+                        <li class="jss">Selenium</li>
+                    </ul>
+                </div>
+                <div class="skillscont">
+                    <div class="lang-heading"><img src="images/circle.png" alt="">
+                        <h3 class="iconic"></h3>Skills
+                    </div>
+                    <ul class="langlist">
+                        <li class="jss">Codekit</li>
+                        <li class="jss">Github</li>
+                        <li class="jss">Codepen</li>
+                        <li class="jss"> Gitlab</li>
+                        <li class="jss"> Terminal</li>
+                    </ul>
+                </div>
             </div>
-        </div>
 
     </section>
 
+    <form action="https://formspree.io/f/mlevdeoa" method="post" class="contact-form">
+        <div class="interest">
+            <h5>I'm always interested in hearing about new projects,
+                so if you'd like to chat please get in touch.</h5>
+        </div>
+        <div class="all">
+            <div><input type="text" placeholder="Full name" maxlength="30" class="inputs" name="user-name" required>
+            </div>
+            <div><input type="email" placeholder="Email" class="inputs" name="e-mail" required></div>
+            <div class="text-cont"><textarea name="message" placeholder="Enter your text here" maxlength="500"
+                    class="ta" required></textarea> </div>
+            <button type="submit" id="get-me" required class="btz">Get in touch</button>
+        </div>
+    </form>
+
+    <div class="border"></div>
 
+    <footer class="feet">
+        <ul class="footer-icons">
+            <li class="ikes" id="foots"><a href=""><i class="fa-brands fa-github"></i></a></li>
+            <li class="ikes" id="foots"><a href=""><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+            <li class=ikes id="foots"><a href="f"><i class="fab fa-angellist"></i></a></li>
+            <li class="ikes" id="foots"><a href=""><i class="fa-brands fa-twitter"></i></a></i></li>
+        </ul>
```

Comment:
- [ ] [OPTIONAL] Please try to style these footer social media icons in order to match the ones we have in the Figma design. As a developer, attention to detail is a great skill that will make you outstanding. Please take a look at the comparison below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/186247726-a06e5df2-5d5b-4175-8a30-8c7863a33343.png)

**Figma**
![image](https://user-images.githubusercontent.com/90222110/186247877-4c3a78e6-4e36-4724-8fb3-6e4791186e8d.png)